### DISCUSSION TITLE
How generate the 2FA authentication by using Postman

### DISCUSSION THREAD

**Discussion by Ankit8420** [octokit/octokit.js#1829] [Q&A]
I am trying to do the 2FA by using the postman.
This is a POST request
Url: https://api.github.com/authorization
in authorization, i have pass my username and password.
I am trying to access my account having 2FA using Postman.
Please help
Thanks

**Comment by gr2m** [comment]
Hey @Ankit8420, I'm afraid I cannot help with Postman, it's out of scope for the JavaScript Octokit SDK, and I don't use the app myself.

Also note that using username/password for authentication against the API is deprecated and will be removed in future.

If you want to use Basic auth in JavaScript code, you can use this package: https://github.com/octokit/auth-basic.js
